tasks/config/licenses.js

```diff
@@ -48,7 +48,9 @@ module.exports = function () {
         'flatten@0.0.1': ['MIT'],
         'indexof@0.0.1': ['MIT'],
         'jsonify@0.0.0': ['Public domain'],
+        'regexp-quote@0.0.0': ['MIT'],
         'ripemd160@0.2.0': ['MIT'],
+        'sanitize-html@1.14.1': ['MIT'],
```

Comment:
Why does `sanitize-html` need to be in here?  That package seems to publish its MIT license appropriately in both package.json and a LICENSE file, so I would have assumed the license checker would pick it up.